src/tools/rbtree.c

```diff
@@ -31,13 +31,18 @@ struct rbtree {
     rbnode *root;
     const char* name;
     bool is_unstable;
+    // Cache 
+    rbnode *righter; 
+    rbnode *lefter;
```

Comment:
Refine the naming scheme to reflect leftmost and rightmost. Also, improve the comments.

Reply:
Place TODO comments for enforcing the consistent naming.